docs/2.build/2.smart-contracts/anatomy/collections.md

```diff
@@ -15,15 +15,19 @@ You can choose between two types of collections:
 
 Understanding how the contract stores and loads both types of collections is crucial to decide which one to use.
 
-:::tip
+<details>
 
-Use native collections for small amounts of data that need to be accessed all together, and SDK collections for large amounts of data that do not need to be accessed all together
+<summary> Native vs SDK Collections </summary> 
 
-:::
+Use native collections for small amounts of data that need to be accessed all together, and SDK collections for large amounts of data that do not need to be accessed all together.
+If your collection has up to 100 entries, it's acceptable to use the native collection, as it might be simpler since you don't have to manage prefixes as we do with SDK collections.
```

Comment:
```suggestion

If your collection has up to 100 entries, it's acceptable to use the native collection, as it might be simpler since you don't have to manage prefixes as we do with SDK collections.
```